domains/emissions/Emissions.API/API/Models/DeclarationProduction.cs

```diff
@@ -1,48 +1,48 @@
-using System.Text.Json.Serialization;
-
-namespace API.Models;
-
-public class DeclarationProduction
-{
-    public string Help { get; }
-
-    public bool Success { get; }
-
-    public Result Result { get; }
-
+using System.Text.Json.Serialization;
+
+namespace API.Models;
+
+public class DeclarationProduction
+{
+    public string Help { get; }
+
+    public bool Success { get; }
+
+    public Result Result { get; }
+
     public DeclarationProduction(string help, bool success, Result result)
     {
         Help = help;
         Success = success;
         Result = result;
     }
-}
-
-public class Field
-{
-    public string Type { get; }
-
-    public string Id { get; }
-
+}
+
+public class Field

```

Comment:
Best practice in C# is to have one class per file.

Reply:
Refactored